python/cudf/cudf/core/frame.py

```diff
@@ -2567,11 +2498,9 @@ def _colwise_binop(
             # are not numerical using the new binops mixin.
 
             outcol = (
-                left_column.binary_operator(fn, right_column, reflect=reflect)
-                if right_column is not None
-                else column_empty(
-                    left_column.size, left_column.dtype, masked=True
-                )
+                getattr(operator, fn)(right_column, left_column)
+                if reflect
+                else getattr(operator, fn)(left_column, right_column)
```

Comment:
I would avoid the repeated logic here. Add a line like `lhs, rhs = (self, other) if not reflect else (other, self)` above and then call `outcol = getattr(operator, fn)(lhs, rhs)`.

Reply:
Already done in a branch I've made off of this one, so let's leave this for now.